src/spyglass/position/v1/position_trodes_position.py

```diff
@@ -382,8 +396,31 @@ def make_video(
             output_video_filename, fourcc, frame_rate, frame_size, True
         )
 
+        if test_mode or truncate_data:
```

Comment:
@samuelbray32 I was able to replicate that error I previously saw with `fill_nan`. It looks like the `minirec` sample has a much longer position_time than reflected in the centriods that get to this point. Is that a byproduct of how they were truncated? If we can chop at the source, it might be worth doing to avoid these workarounds for testing

Reply:
That would make sense.  The conversion was done by truncating the rec file, but the position data comes from a different data file.  I could make a version with less position data.

In general I don't know that it's safe to expect that the position data in the nwb will fully contained in the epoch interval.  Is the issue with any extension past or just the magnitude in this case?

Reply:
I think it's just the magnitude

Reply:
I'm using the one present on `trodes_to_nwb` box folder. Maybe that one is out of date?

Reply:
@CBroz1 Following up, I downloaded and opened the `minirec` file from box and the spatial times, epoch times and video times start and stop matched within ~1s.   

Is it possible that this is running and hitting error with `mediumnwb`?

Reply:
No - the tests I'm running are def `minirec`. It's a fresh container with only `minirec` available. Hitting this error previously, I saw a discrepancy of ~1500 vs ~270

Reply:
Another thought: what `trodes_pos_params_name` is being populated here?  When trying to test this table myself today I found it gave a similar looking error in cases where the position data was upsampled.

Reply:
That may be it - this mismatch may upsampling causing this. My pytest upsampling is set to 500, which would account for the magnitude difference. I think that's the bug then - that upsampled data could get passed to `fill_nan` without accounting for index

Reply:
Opened #989 to address in future PR